code-scanning/properties/phpmd.properties.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "PHPMD",
+  "description": "A spin-off project of PHP Depend and aims to be a PHP equivalent of the well known Java tool PMD.",
+  "iconName": "phpmd",
```

Comment:
It would be hard for users to read the text in logo. Do you think it can be different?

Reply:
<img width="410" alt="Screenshot 2022-06-03 at 12 06 37 PM" src="https://user-images.githubusercontent.com/25999504/171800543-1006d2cb-917c-47e7-8c97-3a8863f922ba.png">
Logo is small when showing the template

Reply:
I have created a issue asking the owners if they will draft a better fit logo or keep using the current one.

Reply:
Nice! I have got replied form repo owners and used the one instructed, the trashcan logo without text.
Updated the PR, please review.